Makefile

```diff
@@ -216,23 +228,23 @@ certs: ## Generate TLS certificates
 	cp ${CERTS_DIR}/server/ee.key ${CERTS_DIR}/server.key
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Docker Helper Targets                                                                                           #
+# Container Image Helper Targets                                                                                  #
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-build-docker: # Build agent docker image for NGINX Plus, need nginx-repo.crt and nginx-repo.key in build directory
-	@echo Building Docker; \
-	DOCKER_BUILDKIT=1 docker build -t ${DOCKER_TAG} . \
-		--no-cache -f ./scripts/docker/nginx-plus/${OS_RELEASE}/Dockerfile \
+image: ## Build agent container image for NGINX Plus, need nginx-repo.crt and nginx-repo.key in build directory
+	@echo Building image with $(CONTAINER_CLITOOL); \
+	$(CONTAINER_BUILDENV) $(CONTAINER_CLITOOL) build -t ${IMAGE_TAG} . \
+		--no-cache -f ./scripts/docker/${OS_RELEASE}/Dockerfile \
```

Comment:
The path for the dockerfile should be `./scripts/docker/nginx-plus/${OS_RELEASE}/Dockerfile`
We recently moved some dockerfiles to a `nginx-plus` folder under `scripts/docker/`

Reply:
@dhurley my bad, fixed.